### ISSUE TITLE
Thoughts on XP component 

### ISSUE THREAD

**Issue by ALIENQuake** [Black-Wyrm-Lair/Drizzt-Saga#16]
Hi,
I always thought that this component is now completely unnecessary:

- no longer depends on the main component but it can't be installed without it on classic bg2
```
Installing [Raise the XP cap] [v3.00]
Copying and patching 1 file ...
ERROR: error loading [DRIZZTSAGA\Language\english\%SrcFolder%]
Stopping installation because of error.
Stopping installation because of error.

ERROR Installing [Raise the XP cap], rolling back to previous state
Will uninstall   0 files for [DRIZZTSAGA/DRIZZTSAGA.TP2] component 2.
Uninstalled      0 files for [DRIZZTSAGA/DRIZZTSAGA.TP2] component 2.
ERROR: Unix.Unix_error(20, "stat", "DRIZZTSAGA\\Language\\english\\%SrcFolder%")
Please make a backup of the file: debugs/SETUP-DRIZZTSAGA.DEBUG and look for support at: flysoup, http://forums.blackwyrmlair.net/index.php?showforum=103
Automatically Skipping [Raise the XP cap] because of error.
Using Language [English]
```
- it overlaps with similar components from BG2Tweaks/BGT Tweaks/EET Tweaks and even Scales of Balance.

More doesn't always seem better, my proposal is to deprecate this component and instruct players to use well-maintained and proven components from other 'tweaks' mods.

**Comment by Roberciiik** [comment]
What do you mean by "installation on classic BG2"? Drizzt Saga should be installed on Tutu or BGT on classic engine. I think we could deprecate this as well as change it to require the first component (I feel like nobody would install Drizzt-Saga only for that tweak).

**Comment by ALIENQuake** [comment]
I'm referring to this line: https://github.com/Black-Wyrm-Lair/Drizzt-Saga/blob/master/drizztsaga/drizztsaga.tp2#L198
```
///////////////////////////////////////////////////////////////////
//// XP cap patching - now an optional component //////////////////
///////////////////////////////////////////////////////////////////

BEGIN @1507  // Raise the XP cap
DESIGNATED 2
//REQUIRE_COMPONENT "drizztsaga.tp2" "0"  @1503
REQUIRE_PREDICATE NOT GAME_IS ~BGT~ @1506
```
The requirements for the main component were disabled. This seems to be correct because there is no logical reason why 'XP cap modification' should require the installation of some additional component(apart from needed files). However, it also means that this component could be installed directly on any game, BG1, BG2, BG1EE, BG2EE, EET etc And if that is so, it's better for people to use XP components that are actively maintained.

The XP cap was not meant to be altered by The Drizzt Saga. However, the excessive XP rewards have led to the 'rebalanced' elements being implemented. With the 'rebalanced' elements in place, it's time to move on.

If there won't any objection, I will deprecate the component.

**Comment by Roberciiik** [comment]
Yup, go for it! 
You could also mention it in the readme that people should use well-maintained components from various tweaks, instead of this one.

Regarding the error from the first post, what was your setup? Just classic BG2 and you installed this component or something else?

**Comment by ALIENQuake** [comment]
Clean, classic BG2 only.

**Comment by Roberciiik** [comment]
I've fixed the faulty component in the #17. There was missing load of cpmvars for clean BG. Nevertheless, please continue with deprecating this component.
